### PR TITLE
Parallelize jobs in travis

### PR DESCRIPTION
**Pull request by tzimisce012**
Related to #75

### CODE REVIEW

**Comment by JesusMtnez** [comment]
From my point of view, this is a partial fix, but it introduces one problem.
Even if the `build` step fails, the docker artifact is released. 

Looking at the logs, the multi-step build using `scala-sbt` to build the fatjar for the server is taking too long because it has to compile `sbt`, `scala` and the whole project again. IMHO, I think `universal:packageBin` step should be done using the cache from the previous build success, to avoid duplicating efforts. 

What do you think?

**Comment by tzimisce012** [comment]
If the `build` step fails, the docker artefact won't be released. 

The problem with Travis is that they don't cache docker layers, so every time it builds, it is like a fresh docker install.

**Comment by JesusMtnez** [comment]
> If the build step fails, the docker artifact won't be released.

:+1: Then no problem on that one

> The problem with Travis is that they don't cache docker layers, so every time it builds, it is like a fresh docker install.

That's why I think that instead of using `sbt-scala` image and a multi-stage `Dockerfile`, we could take advantage of all elements that are already installed in `build` step and also run `universal:packageBin` in `build` step. Then we could cache the artifact for the docker-build step. That way, we would not need `sbt-scala` to recompile everything.

**Comment by tzimisce012** [comment]
Yep, I guess I can compile outside of docker and then use the compiled jar to create the docker image, instead of compiling inside of docker

File: project/plugins.sbt [hunk 1]
```diff
@@ -6,4 +6,3 @@ addSbtPlugin("com.47deg"                 % "sbt-microsites"      % "1.1.0")
 addSbtPlugin("com.timushev.sbt"          % "sbt-updates"         % "0.5.0")
 addSbtPlugin("com.typesafe.sbt"          % "sbt-native-packager" % "1.6.1")
 addSbtPlugin("com.eed3si9n"              % "sbt-buildinfo"       % "0.9.0")
-addSbtPlugin("com.47deg"                 % "sbt-microsites"      % "1.1.0")
```
Comment (JesusMtnez) [inline project/plugins.sbt]: Why removing `sbt-microsites`? We are using it :thinking:

**Comment by tzimisce012** [comment]
Added cache for docker layers. Now the CI is using between 5 and 7 minutes

**Pull request merged**